spec/systems/users/view_sign_up_page_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'View sign up page', type: :system do
```

Comment:
While it seems like we're testing Devise implementation, these tests still offer some values. They establish our requirements, safe-guard future developments, i.e. customize sign up view

Reply:
Thank for teh clarification